queries/zig/highlights.scm

```diff
@@ -2,11 +2,7 @@
   (container_doc_comment)
   (doc_comment)
   (line_comment)
-] @comment @spell
```

Comment:
so spelling will be removed with this PR? Intention is to let nvim spell on `@string`/`@comment`?

Reply:
Nvim doesn't bundle zig parser or queries, so if it's dropped here, there will be no spelling.

Reply:
I thought nvim had a fallback to `@string`/`@comment` if the query doesn't provide any `@spell`

Reply:
I don't see how that bit could have been removed as I didn't touch it, but it's fixed now.